llvm/include/llvm/Transforms/Utils/SimplifyIndVar.h

```diff
@@ -47,10 +49,12 @@ class IVVisitor {
 
 /// simplifyUsersOfIV - Simplify instructions that use this induction variable
 /// by using ScalarEvolution to analyze the IV's recurrence.
```

Comment:
Add comment on what the new return value represents.

Reply:
Added, thanks!